README.md

```diff
@@ -1,2 +1,16 @@
 # linkage
 Implementation of the Linkage algorithm that pairs with the Tokenizer
+
+## Installation
+
+Create a virtual environment:
+
+    python3.11 -m venv venv
```

Comment:
Are we pinned to 3.11 version somehow? If not, we should rename this to use just `python`.